iot_verifier/src/main.rs

```diff
@@ -65,84 +65,119 @@ impl Server {
         // Install the prometheus metrics exporter
         poc_metrics::start_metrics(&settings.metrics)?;
 
-        // configure shutdown trigger
-        let (shutdown_trigger, shutdown) = triggered::trigger();
-        let mut sigterm = signal::unix::signal(signal::unix::SignalKind::terminate())?;
-        tokio::spawn(async move {
-            tokio::select! {
-                _ = sigterm.recv() => shutdown_trigger.trigger(),
-                _ = signal::ctrl_c() => shutdown_trigger.trigger(),
-            }
-        });
-
         // Create database pool and run migrations
         let pool = settings.database.connect(env!("CARGO_PKG_NAME")).await?;
         sqlx::migrate!().run(&pool).await?;
 
         telemetry::initialize(&pool).await?;
 
+        let (file_upload, file_upload_server) =
+            file_upload::FileUpload::from_settings_tm(&settings.output).await?;
```

Comment:
_tm stands for `taskmanager` so that is the link...but the `_tm` functions are all temporary.  Once all oracles are update to use taskmanager these will be renamed to the regular version ie from_settings_tm will become from_settings, new_tm will become new